hearth/lib/hearth/request.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'stringio'
```

Comment:
technically we don't require stringio here anymore since we aren't defaulting to it.